packages/appChrome/components/SidebarSubMenuItem.tsx

```diff
@@ -64,9 +64,13 @@ class SidebarSubMenuItem extends React.PureComponent<
         )]: isActive
       }
     );
+    const dataCy = [
```

Comment:
I can't see the data-cy for this element.

Reply:
works for me!

![Storybook 2019-07-22 10-49-05](https://user-images.githubusercontent.com/300861/61619035-5beab800-ac6e-11e9-8a6d-273f79a585c8.png)